core/sqf/sqenvcom.sh

```diff
@@ -505,23 +505,6 @@ EOF
   APACHE_HIVE_HOME=$HIVE_HOME
   export HIVE_CNF_DIR=$HIVE_HOME/conf
 
```

Comment:
I think not, since the local_hadoop case is the first case in this far above (line262).  The else case seems to be for vanilla hadoop or the build-only case when there is no distro at all.